locations/spiders/olive_young_kr.py

```
@@ -0,0 +1,79 @@
+HEADERS = {
+    "Accept": "application/json",
+    "Origin": "https://www.oliveyoung.co.kr",
+    "Referer": STORE_FINDER_URL,
+    "X-Requested-With": "XMLHttpRequest",
+}
+CENTROID_OFFSET_BEARINGS = (45, 135, 225, 315)
+
+
+class OliveYoungKRSpider(Spider):
+    name = "olive_young_kr"
+    item_attributes = {"brand_wikidata": "Q55643246"}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)